app/client/keybase/debug/keystore.go

```diff
@@ -0,0 +1,134 @@
+package debug
+
+import (
+	"errors"
+	"fmt"
+	"github.com/pokt-network/pocket/app/client/keybase"
+	"github.com/pokt-network/pocket/shared/crypto"
+	"gopkg.in/yaml.v2"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+const (
+	validatorNumber    = 999
+	debugKeybaseSuffix = "/.pocket/keys"
+)
+
+var (
+	DebugKeybasePath string
```

Comment:
s/DebugKeybasePath/debugKeybasePath
```suggestion
         // TODO: Allow users to override this value via `datadir` flag
	debugKeybasePath string
```